test/models/viewmodels/notificationspage/NotificationPageSingleElementFactorySpec.scala

```diff
@@ -40,17 +42,19 @@ class NotificationPageSingleElementFactorySpec extends WordSpec with MustMatcher
   private val formatter = DateTimeFormatter.ISO_OFFSET_DATE_TIME.withZone(ZoneId.systemDefault())
   private val testTimestamp = ZonedDateTime.parse(testTimestampString, formatter).toInstant
 
-  private val crcCodeKeyFromDecoder = CrcCode.Success
-  private val roeKeyFromDecoder = RoeCode.DocumentaryControl
-  private val soeKeyFromDecoder = SoeCode.DeclarationAcceptance
+  private val crcKeyFromDecoder = CRCCode.Success
+  private val icsKeyFromDecoder = ICSCode.InvalidationByCustoms
+  private val roeKeyFromDecoder = ROECode.DocumentaryControl
+  private val soeKeyFromDecoder = SOECode.DeclarationAcceptance
```

Comment:
`ducrSoeKeyFromDecoder`, below we have `mucr` to have consistent naming

Reply:
Renamed.